pyigm/surveys/lls_literature.py

```diff
@@ -0,0 +1,1078 @@
+"""
```

Comment:
Unfortunately not.

On 03/01/2016 14:03, J. Xavier Prochaska wrote:
> In pyigm/surveys/lls_literature.py
> <https://github.com/pyigm/pyigm/pull/5#discussion_r48693390>:
>
>> @@ -0,0 +1,1078 @@
>> +"""
>
> Yes, it is. I think I updated it as we went to clean those
> issues. Can you remember any specific one so that
> I can double check?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyigm/pyigm/pull/5/files#r48693390>.Web Bug from
> https://github.com/notifications/beacon/AL4mf72iixqlFyYzf4rzBW_PVHC6e8sOks5pWSHMgaJpZM4G4tqP.gif
>

-- 
Michele Fumagalli
Durham University
michele.fumagalli@durham.ac.uk
www.michelefumagalli.com